components/LoginModal.js

```diff
@@ -0,0 +1,99 @@
+import s from 'styled-components'
+
+import { Icon } from './common/Icon'
+import { ALLBIRDS_GRAY, LIGHT_GRAY, MEDIUM_GRAY, DARK_GRAY } from '../constants/colors'
+import { BORDER_RADIUS_LG, MD, SM, mediaMaxWidth } from '../constants/measurements'
+import { LOGIN_URL } from '../utils'
+
+const Logo = s.img`
+  width: 100px
+  transform: scale(1);
+  margin-top: 12%;
+`
+
+const ModalWrapper = s.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+  z-index: 1002;
+  text-align: center;
+`
+
+const ModalCard = s.div`
+  border-radius: ${BORDER_RADIUS_LG};
+  border: 0 !important;
+  box-shadow: none !important;
+  height: auto;
+  width: 35%;
+
+  ${mediaMaxWidth(MD)} {
+    width: 50%;
+  }
+
+  ${mediaMaxWidth(SM)} {
+    width: 95%;
+  }
+`
+
+const ModalTitle = s.h1`
+  color: ${DARK_GRAY};
+  font-size: 2rem;
+  font-weight: 600;
+  line-height: 1.125;
+  margin: 2% 0;
+`
+
+const ModalContent = s.div`
+  margin: auto;
+  margin-bottom: 10%;
+`
+
+const CloseModalIcon = s(Icon)`
+  position: absolute;
+  right: 20px;
+  top: 20px;
+  cursor: pointer;
+  color: ${LIGHT_GRAY};
+
+  &:hover {
+    color: ${MEDIUM_GRAY};
+  }
```

Comment:
Note that this currently won't actually change the color of the icons, the current way we handle svgs are resistant to simple css `color` rules. :(

Reply:
I'll remove it then, this is probably left over from the font awesome days.